Modelica/Electrical/Polyphase/Basic/Capacitor.mo

```diff
@@ -3,6 +3,8 @@ model Capacitor "Ideal linear electrical capacitors"
   extends Interfaces.TwoPlug;
   parameter SI.Capacitance C[m](start=fill(1, m))
     "Capacitance";
+  SI.Voltage vC[m](start=zeros(m), fixed=fill(false,m))=v "Capacitor voltages"
```

Comment:
```suggestion
  SI.Voltage vC[m](each start=0, each fixed=false)=v "Capacitor voltages"
```
The code works, but I would prefer using `each` to avoid repeating the size.

Reply:
I do not prefer this.

With the `each`, the "natural" edit is to change the value (keeping the each), but from what I understand, the expected thing here is to set an array with different values.

In System Modeler, this variant with `each` will not show an array editor, but default to let you edit the value `0`/`false`.

Reply:
Ok then.